modules/end-user-guide/pages/basic-actions-you-can-perform-on-a-workspace.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: REFERENCE
+:description: Basic actions you can perform on a workspace
+:keywords: running-workspace, stopped-workspace, open-workspace, open-a-workspace, open-a-running-workspace, opening-workspace, opening-a-workspace, opening-a-running-workspace, how-to-open-workspace, how-to-open-a-workspace, how-to-open-a-running-workspace, reopen-workspace, reopen-a-workspace, reopen-a-running-workspace, reopening-workspace, reopening-a-workspace, reopening-a-running-workspace, how-to-reopen-workspace, how-to-reopen-a-workspace, how-to-reopen-a-running-workspace, restart-workspace, restart-a-workspace, restart-a-running-workspace, restarting-workspace, restarting-a-workspace, restarting-a-running-workspace, how-to-restart-workspace, how-to-restart-a-workspace, how-to-restart-a-running-workspace, stop-workspace, stop-a-workspace, stop-a-running-workspace,  stopping-workspace, stopping-a-workspace, stopping-a-running-workspace, how-to-stop-workspace, how-to-stop-a-workspace, how-to-stop-a-running-workspace, start-workspace, start-a-workspace, start-stopped-workspace, start-a-stopped-workspace, starting-workspace, starting-a-workspace, starting-stopped-workspace, starting-a-stopped-workspace, how-to-start-workspace, how-to-start-a-workspace, how-to-start-stopped-workspace, how-to-start-a-stopped-workspace, delete-workspace, delete-a-workspace, deleting-workspace, deleting-a-workspace, how-to-delete-workspace, how-to-delete-a-workspace
+:navtitle: Basic actions you can perform on a workspace
+// :page-aliases:
+
+[id="basic-actions-you-can-perform-on-a-workspace_{context}"]
+= Basic actions you can perform on a workspace
+
+You manage your workspaces and verify their current states in the *Workspaces* page (`{prod-url}/dashboard/#/workspaces`) of your {prod-short} dashboard.
+
+After you start a new workspace, you can perform the following actions on it in the *Workspaces* page:
```

Comment:
```suggestion
After you start a new workspace, you can stop, start, reopen, restart, and delete it in the *Workspaces* page (`{prod-url}/dashboard/#/workspaces`) of your {prod-short} dashboard.
```

Reply:
In the openshift-docs repo (not the eclipse-che docs) I see references to `*Workspaces* section` instead of `page`. It might be good to review this and "harmonize" our nomenclature, eventually.

Reply:
Let's look at this: by *Workspaces* page in this sentence I mean the web application GUI of CodeReady Workspaces.

I've jus tried searching for *Workspaces*:
https://github.com/openshift/openshift-docs/search?q=*Workspaces*
And I assume this is what you meant:
https://github.com/openshift/openshift-docs/blob/0fd09865e36c7eb97a8230a661632f62866a30c5/modules/op-constructing-pipelines-using-pipeline-builder.adoc
And this result appears to be describing YAML content sections.
If so, then we needn't worry about harmonizing.

BTW, I've discovered some outdated content in the search results and already opened a Jira issue in our docs for our team to get in touch with the OpenShift writers. This is outdated: https://github.com/openshift/openshift-docs/blob/b0199378013f0684a693f66c5bdb5dd430642c41/modules/codeready-workspaces.adoc

Reply:
@rolfedh, i appreciate your suggestion. The problem for me here and the reason why I added the table is the fact that the current GUI implementation unfortunately needs explaining. The second reason for the table is indication of the state of the workspace: some actions are performed on a running workspace, while one of the actions (starting a stopped workspace) is performed on a stopped workspace, while the last action (deleting) is applicable to both running and stopped workspaces. I realize this info may appear trivial, but for example restarting a running workspace and starting a stopped workspace (which can also be defined as restarting a workspace) are two different actions that are taken using different UI elements.